Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -644,6 +733,10 @@ public static void GetDeactivatedAssetNotification(Action<LootLockerDeactivatedA
             LootLockerAPIManager.GetDeactivatedAssetNotification(onComplete);
         }
 
+        /// <summary>
+        /// 5 minutes after calling this endpoint you should issue a call to the Player Asset Notifications call to get the results of the migrated DLC, if any.
```

Comment:
How come they should call PlayerAssetNotification and not the GetDLCMigrated from below? Different levels?

Reply:
You can call any of them, depends on what you want to do with the information. GetDLCMigrated returns just a simple list of the DLC as so;  "dlcs": ["257300", "257301", "257302", "433770"]
PlayerAssetNotification returns a more detailed list of the assets and will also include any other assets that have been added during the time. I'll write something about it 👍